libcxxabi/cmake/Modules/HandleLibC.cmake

```diff
@@ -0,0 +1,39 @@
+#===============================================================================
```

Comment:
Did you really intend to add this file? Aren't those redundant with the ones in `runtimes/cmake/Modules`?

Reply:
These are no longer used, removed.